frontend/src/components/widgets/inputs/ColorInput.vue

```diff
@@ -84,6 +84,7 @@ export default defineComponent({
 	props: {
 		value: { type: String as PropType<string | undefined>, required: true },
 		label: { type: String as PropType<string>, required: false },
+		canSetTransparent: { type: Boolean as PropType<Boolean>, required: false, default: true },
```

Comment:
I don't really know why typescript has both `Boolean` and `boolean` but I think GitHub wants this one to be `{ type: Boolean as PropType<boolean> ...` (as for the disabled prop below)

Reply:
Thanks